pages/auvtestbed.html

```diff
@@ -0,0 +1,113 @@
+
+    <link rel="stylesheet" type="text/css" href="../css/bootstrap.css">
+    <link rel="stylesheet" type="text/css" href="../css/component.css" />
+
+    <script src="../js/modernizr.custom.js"></script>
+    <script src="../js/jquery.js"></script>
+    <script src="../js/bootstrap.js"></script>
+    <script src="../js/func.js"></script>
+
+</head>
+
+<body>
+    <div id="navbar">
+        <script type="text/javascript">
+        $("#navbar").load('navbar.html', function() {
+            changeActive(5);
+        });
+        </script>
+    </div>
+
+    <div class="container" style="margin-top:70px;">
+        <h1 style="text-align:center; top:0px; left:0px;">OVERVIEW</h1>
+            <div class="row">
+                <div class="col-md-1"></div>
+                <div class="col-md-8">
+                    <div class="well well-lg">
+                        <div style="text-align:center">
+                        <img src="../resources/AUV.jpg" class="img-rounded" height="400" width="660"/ alt="" style="margin-top:20px;">
+                        </div>
+                        <br>
+                        <br>
+                        <br>
+                        Indian Institute Of Technology Kharagpur along with CSIR Lab, CMERI Durgapur developed an autonomous underwater vehicle (AUV).  
+                        An autonomous underwater vehicle is a robot which travels underwater. Sometimes called Unmanned Underwater Vehicles, 
+                        these devices are powered by batteries and can operate in water as deep as 150 meters. 
+                        It is based an advances in propulsion systems and power source technology give these robotic submarines extended endurance in both time and distance.
+                        The Intelligent Systems and Robotics Lab, CAD-CAM laboratories of Mechanical Engineering Department is developing a prototype AUV test bed for  R&D in Underwater robotics.
+
+                        This is under a project being carried out with support from Ministry Of Earth Sciences, Government  Of INDIA under the Ocean and Atmospheric Science and Technology Cell of IIT-Kharagpur.
+                        <br>
+                        <br>
+                        <b>Following Groups are Involved</b>
+                        <br>
+                        <br>
+                        a) Hydrodynamics and Design group, headed by Principal Investigator ,Prof.Debarata Sen of Ocean Engineering And Naval Architecture
+                        <br>
+                        b )Prototyping , Navigation and Control group, headed by Co-Principal Investigator, Dr.C.S.Kumar Of Mechanical Engineering Department.
+                        <br>
+                        c )Communication systems group,  headed by  Co-Principal Investigator,   Prof D.Kastha of    Electrical Engineering Department
+                        <br>
+                        The Groups Work closely with Corresponding groups at CMERI Durgapur.
```

Comment:
`These groups work closely with their corresponding counterparts at the CMERI Durgapur`